api/pkg/service/auth/auth_http_test.go

```diff
@@ -23,25 +23,13 @@ import (
 
 	"github.com/ikawaha/goahttpcheck"
 	"github.com/stretchr/testify/assert"
-	goa "goa.design/goa/v3/pkg"
-	"gopkg.in/h2non/gock.v1"
-
 	"github.com/tektoncd/hub/api/gen/auth"
 	"github.com/tektoncd/hub/api/gen/http/auth/server"
 	"github.com/tektoncd/hub/api/pkg/testutils"
+	goa "goa.design/goa/v3/pkg"
```

Comment:
is this part of `go fmt` ?

Reply:
yes we used to keep a line between 3rd party imports and internal imports but go fmt doesn't so that
so removing it in some file in this patch

Reply:
gorm patch will update in remaining files

Reply:
we should rely on a single tool to do this and not mix it up. `go fmt` is the acceptable tool. Lets please stick to that 🙏 . Could you please revert this and get `go fmt` do its work ?